Jenkinsfile

```diff
@@ -392,8 +389,64 @@ pipeline {
               }
               build(job: 'apm-server/apm-hey-test-benchmark', propagate: true, wait: true,
                     parameters: [string(name: 'GO_VERSION', value: '1.12.1'),
-                                string(name: 'STACK_VERSION', value: "${env.GIT_BASE_COMMIT}"),
-                                string(name: 'APM_DOCKER_IMAGE', value: "${env.DOCKER_IMAGE}")])
+                                 string(name: 'STACK_VERSION', value: "${env.GIT_BASE_COMMIT}"),
+                                 string(name: 'APM_DOCKER_IMAGE', value: "${env.DOCKER_IMAGE}")])
+            }
+          }
+        }
+        stage('Package') {
```

Comment:
The previous stage was `release`, in this case, it has been changed to `Package` that contains another two nested stages:
- `Package`, the same logic that the one that used to be in addition to the docker image generation.
- `Publish`, the same logic that the `post` action used to be but moved to a step then this will be triggered only for master/tags